stack.yaml

```diff
@@ -70,24 +70,32 @@ extra-deps:
 - repline-0.2.1.0
 - git: https://github.com/target/row-types
   commit: 1e8d5e084ffd46f6c7842826a1f62c60820885df
+  # nix-sha256: 0ly5m4r8wkm8gdqyrqzsjfmp189yxsd4qp0zi3idrbgfaf45sk9k
```

Comment:
Where do these come from? How are they generated?

Reply:
`nix-prefetch-git --quiet https://github.com/target/row-types 1e8d5e084ffd46f6c7842826a1f62c60820885df | jq .sha256` :'(

@erikd is working on some cardano-repo-tool integration which can compute the hashes without nix. We currently only have python script to inject them into cabal.projects (https://github.com/input-output-hk/cardano-explorer/pull/224/files#diff-b077822def5171a1a687e7bc9d0b9558), but hopefully this could be extended to stack.yaml files as well.

Reply:
This needs to be written down somewhere in our documentation. I'll add it.